src/Demos/HSP.lagda

```diff
@@ -1,9 +1,8 @@
 \section{Introduction}
 The \agdaalgebras library \cite{ualib_v2.0.1} formalizes the foundations of universal algebra
 in intensional Martin-Löf type theory (\mltt) using \agda~\cite{Norell:2007,agdaref}.
-The library includes a wide collection of definitions and verified theorems that faithfully codify
-classical, set-theory-based universal algebra and equational
-logic.
+The library includes a wide collection of definitions and verified theorems originated in classical
+set-theory based universal algebra and universal logic, but have been adapted to fit within MLTT.
```

Comment:
again, I prefer "euqational logic" over "universal logic"

Reply:
Oh, I now see that I was commenting on HSP.tex changes... oops!  We really should have left the (automatically generated) HSP.tex file out of the repo.  Let's remove it soon!